opencensus/stats/internal/stats_manager_test.cc

```diff
@@ -28,14 +28,14 @@ constexpr char kFirstMeasureId[] = "first_measure_name";
 constexpr char kSecondMeasureId[] = "second_measure_name";
 
 MeasureDouble FirstMeasure() {
-  static MeasureDouble measure = MeasureRegistry::RegisterDouble(
-      kFirstMeasureId, "ops", "Usage of resource 1.");
+  static MeasureDouble measure =
+      MeasureDouble::Register(kFirstMeasureId, "ops", "Usage of resource 1.");
```

Comment:
IMO use "1" instead of "ops" to match the standard.

Reply:
Done.